project/Dependencies.scala

```diff
@@ -47,7 +47,7 @@ object Dependencies {
 
   val akkaVersion = "2.5.3"
```

Comment:
2.5.8?

Reply:
Yes, we could update it. But should probably be done in a separate PR.